src/MetricScene/PanelMenu/actions/ExploreAction.ts

```diff
@@ -31,7 +32,11 @@ export class ExploreAction {
     return {
       text: 'Explore',
       iconClassName: 'compass',
-      onClick: () => exploreUrl?.then((url) => url && window.open(url, '_blank')),
+      onClick: () => exploreUrl?.then((url) => {
+        if (url) {
+          window.open(`${config.appSubUrl}${url}`, '_blank');
```

Comment:
Looking at some tests in `grafana/grafana` (like [this one](https://github.com/grafana/grafana/blob/84a2f41016c01aa5209346e9531354abd1aa4629/packages/grafana-data/src/utils/location.test.ts#L37)), it looks like the `appSubUrl` is expected to have a leading `/` but not a trailing `/`. If that's correct, I believe we need a `/` between the `appSubUrl` and `url`.

Reply:
@NWRichmond here's the actual leading `/` in the scenes repo from `getExploreUrl`

https://github.com/grafana/scenes/blob/badacd884623a85905be1a40655d69594fd9f27c/packages/scenes/src/utils/explore.ts#L62-L64